openrtb/osversion.go

```diff
@@ -1,17 +1,11 @@
 package openrtb
 
 import (
-	"strconv"
+	"github.com/Vungle/vungo/openrtb/util"
```

Comment:
This package name is going to be confusing for people, because we have `openrtbutil` and then `util`.  Maybe, simply call this package `github.com/Vungle/vungo/vercmp`?